crates/rpc-types-mev/src/common.rs

```diff
@@ -54,9 +54,10 @@ pub struct Privacy {
     /// Hints on what data should be shared about the bundle and its transactions
     #[serde(skip_serializing_if = "Option::is_none")]
     pub hints: Option<PrivacyHint>,
-    /// The addresses of the builders that should be allowed to see the bundle/transaction.
+    /// Names of the builders that should be allowed to see the bundle/transaction.
+    /// https://github.com/flashbots/dowg/blob/main/builder-registrations.json
```

Comment:
```suggestion
    /// <https://github.com/flashbots/dowg/blob/main/builder-registrations.json>
```

Reply:
oops, fixed link formatting before and forgot to force push